### PR TITLE
Migrate to new style

### PR DESCRIPTION
**Pull request by qsollet**
Included the scoreboard branch as both of them need to be merge at the same time to avoid breaking the style.

### CODE REVIEW

**Comment by qsollet** [comment]
Linked to issue #960 and #959

**Comment by CelineBoudier** [comment]
It looks good!

File: game/static/game/js/scoreboard.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
+/*
+ Code for Life
+
+ Copyright (C) 2016, Ocado Innovation Limited
```
Comment (CelineBoudier) [inline game/static/game/js/scoreboard.js]: 2018

**Comment by CelineBoudier** [comment]
A few codeclimate issues but I think they were here before? @qsollet @j4mesholland

**Comment by CelineBoudier** [comment]
Any opinion on this @mrniket ?

File: game/views/scoreboard.py [hunk 2]
```diff
@@ -118,11 +119,20 @@ def classes_for(user):
 
 
 def scoreboard_view(request, form, student_data, headers):
+    episodes = level_selection.fetch_episode_data_from_database(False)
+    ep = {}
```
[the remark this answers is not in the document]
Comment (CelineBoudier) [reply game/views/scoreboard.py]: +1

File: game/static/game/js/scoreboard.js [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+$('.episode').on('change', function() {
+    var check = $(this).find('input').is(':checked')
+    var first = $(this).attr('data-first')
+    var last = $(this).attr('data-last')
+    for (var i = first; i <= last; i++) {
+        $('.level-'+i).find(':checkbox').prop('checked', check).closest('li').toggleClass('checked', check)
+    }
+})
+
+$('.expender').on('click', function() {
```
[the remark this answers is not in the document]
Comment (CelineBoudier) [reply game/static/game/js/scoreboard.js]: +1

File: game/views/scoreboard.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (qsollet) [reply game/views/scoreboard.py]: Done!

File: game/static/game/js/scoreboard.js (hunk 3 above)
[the remark this answers is not in the document]
Comment (qsollet) [reply game/static/game/js/scoreboard.js]: Done!

**Pull request merged**